recipes/gpi_core/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.0.0rc" %}
+{% set build_num = "0" %}
+{% set sha256 = "8dc97c93e48f57febc67773107276dbbdadb8c8effa75bea18a475c07266442a" %}
+
+package:
+  name: gpi_core
+  version: {{ version }}
+
+source:
+  url: https://github.com/gpilab/core-nodes/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  folder: repo_contents
+
+build:
+  number: {{ build_num }}
+  detect_binary_files_with_prefix: True
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  
+    - {{ compiler('cxx') }}
+  host:
+    - cytoolz
+    - eigen =3
+    - fftw
+    - gpi >=1.1.4
+    - numpy >=1.11
+    - python
+  run:
+    - {{ pin_compatible('eigen') }}
```

Comment:
Do you need eigen at runtime?

Reply:
It's a runtime requirement for another upstream package (`gpi`), so we do need it but it probably doesn't need to be in this recipe too.